.github/workflows/pre-commit.yml

```diff
@@ -13,17 +13,10 @@ jobs:
   pre-commit:
     runs-on: ubuntu-22.04
     steps:
-<<<<<<< before updating
-      - uses: actions/checkout@v2
-      - uses: actions/setup-python@v4
-        with:
-          python-version: "3.10"
-=======
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v2
```

Comment:
Shouldn't this be v4?

Reply:
I don't know, I used `copier update --UNSAFE`

Reply:
v2 seems correct, looking for other repos